code/modules/unit_tests/unit_test.dm

```diff
@@ -104,6 +104,7 @@ var/global/datum/unit_test_controller/unit_tests = new()
 	var/list/fail_reasons
 
 /datum/unit_test/New()
+	SHOULD_CALL_PARENT(TRUE)
```

Comment:
on further review, I shouldn't review when half-asleep and datum/New() already sets this 🤦